pkg/preflight/execute.go

```diff
@@ -121,7 +126,7 @@ func execute(appID string, sequence int64, preflightSpec *troubleshootv1beta2.Pr
 		KubernetesRestConfig:   restConfig,
 	}
 
-	logger.Debug("preflight collect phase")
+	logger.Info("preflight collect phase")
```

Comment:
Moving this to info cause they show progress when performing an installation, which is critical functionality IMO to always log